### PR TITLE
Handle the clear request for 'Q_SHARED_ALL'

### PR DESCRIPTION
**Pull request by bratashX**
Signed-off-by: Petro Bratash <petrox.bratash@intel.com>

**What I did**
Add handling the clear request for 'Q_SHARED_ALL'

**Why I did it**
On this [PR](https://github.com/Azure/sonic-utilities/pull/1149) added new commands :
```
sonic-clear queue persistent-watermark all
sonic-clear queue watermark all
```
Without these changes, commands will result in
https://github.com/Azure/sonic-swss/blob/master/orchagent/watermarkorch.cpp#L221

**How I verified it**
Run:
```
sonic-clear queue persistent-watermark all
sonic-clear queue watermark all
```
**Details if related**
PR in sonic-utilities:  https://github.com/Azure/sonic-utilities/pull/1149

### CODE REVIEW

**Comment by bratashX** [comment]
@akokhan @neethajohn, please review

**Comment by neethajohn** [comment]
Please add unit tests. Refer tests/test_watermark.py

File: orchagent/watermarkorch.cpp [hunk 1]
```diff
@@ -299,10 +309,14 @@ void WatermarkOrch::init_queue_ids()
         {
             m_unicast_queue_ids.push_back(id);
         }
-        else
+        else if (fv.second == "SAI_QUEUE_TYPE_MULTI")
```
Comment (neethajohn) [inline orchagent/watermarkorch.cpp:312]: should be 'SAI_QUEUE_TYPE_MULTICAST'

**Review by neethajohn — CHANGES_REQUESTED** [review]

File: orchagent/watermarkorch.cpp (hunk 1 above)
Comment (bratashX) [reply orchagent/watermarkorch.cpp]: Thanks, fixed

**Comment by bratashX** [comment]
> Please add unit tests. Refer tests/test_watermark.py

@neethajohn Unit test was added, but test-related on  https://github.com/Azure/sonic-utilities/pull/1149, and will pass after merge this PR. 
Also, we skip test in CI:
https://github.com/Azure/sonic-swss/blob/1d3388100ef7edbc275746136d8a1db0769e652e/tests/test_watermark.py#L245
I can temporarily allow it for verification TC and after test passed changes will be rollback.

**Comment by bratashX** [comment]
@neethajohn, could you please review

**Review by neethajohn — APPROVED** [review]

**Comment by bratashX** [comment]
@neethajohn @lguohan could you please review and merge? Thanks

**Pull request merged**